### PR TITLE
Fix msg function as per roosteds request

### PR DESCRIPTION
**Pull request by renzenicolai**

### CODE REVIEW

File: esp32/modules/easydraw.py [hunk 1]
```diff
@@ -18,12 +18,10 @@ def msg(message, clear=False):
     if clear:
         ugfx.clear(ugfx.WHITE)
         ugfx.string(0, 0, message, "PermanentMarker22", ugfx.BLACK)
-        ugfx.set_lut(ugfx.LUT_FASTER)
-        ugfx.flush()
```
Comment (basvs) [inline esp32/modules/easydraw.py]: If clear is set, there's no ugfx.flush() anymore?

Comment (renzenicolai) [reply esp32/modules/easydraw.py]: That is correct, this makes sure that the title isn't redrawn when the message first appears on screen.

It does however require every app that uses this function to call it twice:

msg("title", True)
msg("message") -> this call redraws the screen

**Comment by Roosted7** [comment]
Sorry about the master merges, but that was necessary to keep the branch compiling...

First of all I've removed the global vars that had no reason for being globals.

Secondly, I've made the system automatically clear itself based on the existence of the line counter var
(This also allows for cleaner and easier code in the programs that use this, see the splash)

Thirdly, I've made the title and reset flags optional - user can choose to use them

Fourthly, I've removed the usage of this screen in the wifi helper - I don't think we should force this...
(Would be nice to make it an option, personally I would like to see the wifi and msg api so easy that people can write their own connection [in 2 lines]...)

Tomorrow I'm planning to rewrite this to a Class of it's own so that it can be imported from different modules and still be used easily concurrently

File: esp32/modules/easydraw.py (hunk 1 above)
Comment (Roosted7) [reply esp32/modules/easydraw.py]: This not necessary anymore, title is optional and handled automatically

**Pull request merged**